packages/commonwealth/webpack/webpack.base.config.js

```diff
@@ -22,6 +22,11 @@ module.exports = {
     chunkFilename: 'js/[name].[chunkhash:8].chunk.js',
   },
   plugins: [
+    new webpack.DefinePlugin({
+      'process.env.MIXPANEL_PROD_TOKEN': JSON.stringify(
```

Comment:
You realize this will expose the prod token on the client, correct?

Reply:
yeah, this is required with how mixpanel works. https://developer.mixpanel.com/reference/project-token